### PR TITLE
Task#186 Added Unit tests for UserConroller

### PR DESCRIPTION
**Pull request by olenanovi**
#186  Added unit tests for all methods in UserController

### CODE REVIEW

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,610 @@
...
+
+            // Act
+            var result = await _userController.ApproverDelete(confirmedId);
+
+            // Assert
+            _loggerService.Verify((x) => x.LogInformation(It.IsAny<string>()), Times.Once);
+            Assert.IsInstanceOf<OkResult>(result);
+        }
+
+        public UserDTO CreateFakeUser()
```
Comment (xfreed) [inline EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:522]: If you have faking function, use private modification.
```suggestion
        private UserDTO CreateFakeUser()
```

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,610 @@
...
+                FirstName = "SomeFirstName",
+                LastName = "SomeLastName",
+                UserProfile = new UserProfileDTO()
+                {
+                    EducationId = 1,
+                    WorkId = 1,
+                },
+            };
+
+        public UserViewModel CreateFakeUserViewModel()
```
Comment (xfreed) [inline EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:535]: If you have faking function, use private modification.
```suggestion
        private UserViewModel CreateFakeUserViewModel()
```

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,610 @@
...
+            => new UserViewModel()
+            {
+                ID = "1",
+            };
+
+        public EditUserViewModel CreateFakeEditUserViewModel()
```
Comment (xfreed) [inline EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:541]: If you have faking function, use private modification.
```suggestion
        private EditUserViewModel CreateFakeEditUserViewModel()
```

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs [hunk 4]
```diff
@@ -0,0 +1,610 @@
...
+                },
+                
+                WorkView = new UserWorkViewModel()
+                {
+                    PlaceOfWorkID = 1,
+                    PositionID = 1,
+                },
+            };
+
+        public UserInfoViewModel CreateFaceUserInfoViewModel()
```
Comment (xfreed) [inline EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:564]: If you have faking function, use private modification.
```suggestion
        private UserInfoViewModel CreateFaceUserInfoViewModel()
```

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs [hunk 5]
```diff
@@ -0,0 +1,610 @@
...
+            => new UserInfoViewModel()
+            {
+                Id = "1",
+            };
+
+        public List<ConfirmedUserViewModel> CreateListOfConfirmedUserViewModels()
```
Comment (xfreed) [inline EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:570]: If you have faking function, use private modification.
```suggestion
        private List<ConfirmedUserViewModel> CreateListOfConfirmedUserViewModels()
```

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs [hunk 6]
```diff
@@ -0,0 +1,610 @@
...
+                    {
+                        User = new UserInfoViewModel()
+                        {
+                            ImagePath = "SomeStringID1",
+                        },
+                    },
+                },
+            };
+
+        public ConfirmedUserViewModel CreateConfirmedUserViewModel()
```
Comment (xfreed) [inline EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:597]: If you have faking function, use private modification.
```suggestion
        private ConfirmedUserViewModel CreateConfirmedUserViewModel()
```

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs (hunk 1 above)
Comment (olenanovi) [reply EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:522]: Done.

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs (hunk 2 above)
Comment (olenanovi) [reply EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:535]: Done.

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs (hunk 3 above)
Comment (olenanovi) [reply EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:541]: Done.

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs (hunk 4 above)
Comment (olenanovi) [reply EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:564]: Done.

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs (hunk 5 above)
Comment (olenanovi) [reply EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:570]: Done.

File: EPlast/EPlast.Tests/Controllers/UserControllerTests.cs (hunk 6 above)
Comment (olenanovi) [reply EPlast/EPlast.Tests/Controllers/UserControllerTests.cs:597]: Done.

**Pull request merged**